flutter_lyra_android/test/flutter_lyra_android_test.dart

```diff
@@ -73,8 +73,8 @@ void main() {
 
       test('$PaymentCancelledByUserExceptionInterface', () {
         final exception = PlatformException(
-          code: 'code',
-          message: errorCodesInterface.paymentCancelledByUser,
+          code: errorCodesInterface.paymentCancelledByUser,
+          message: 'message',
```

Comment:
(It's Lyra, not ReachFive)

If we take a real example for it

`message` would always be "FlutterError"
(since the pigeon upgrade)

You could write that in your message to be closer to reality
But it still show that we can't retrieve a real error message currently

We need to improve `FlutterError` native class for that